eventActions/profanityActions.js

```diff
@@ -44,4 +44,4 @@ class snapshotActions {
 
 }
 
-module.exports = snapshotActions;
+module.exports = profanityActions;
```

Comment:
This should be fixed on newest development branch, shouldn't be necessary to re-push it

Reply:
Don't know why that is even there. Pulled the branch, it persisted. Tried to push and it prompted me to pull. Fixed it